### PR TITLE
LVNODCDXI-39

### PR DESCRIPTION
**Pull request by Kindest13**
stylized authentication page, put css variables in helpers & import them in app.css

### CODE REVIEW

**Review by MaxVoloskiy — DISMISSED** [review]

**Review by MaxVoloskiy — APPROVED** [review]

File: src/client/containers/authentication/authentication.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    filterFields = (arr, status) => (status ? arr.filter(el => status === el.isMember) : arr);
+
+    render() {
+      const { isMember } = this.state;
+      const fildsToRender = this.filterFields(formFieldsInfo, isMember);
+      const formTitle = isMember ? 'Login' : 'Registration';
+      const message = isMember ? 'Dont have an account? Register!' : 'Already have an account? Login!';
+      return (
+        <div className="authentication">
+          <img type="image/svg+xml" src={authImage} alt="CooLEqualizer" className="authImage" />
```
Comment (GeFeSTTS) [inline src/client/containers/authentication/authentication.js]: Please, write understandable alt, like "Equalizer icon".

File: src/client/containers/formComponent/formComponent.css [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
-    outline: none;
-    height: 1rem;
-    padding-left: 0.5rem;
-    color: rgb(0, 255, 255);
+    border-radius: 0.5rem;
+    height: 2rem;
+    color: var(--authInputFont-black);
 }
 
 ::placeholder {
```
Comment (StanislavFedyna) [inline src/client/containers/formComponent/formComponent.css]: Use please full path, like:
`.field input ::placeholder`

**Review by MaxVoloskiy — DISMISSED** [review]

File: src/client/containers/authentication/authentication.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          <div className="form-container">
+            <h1 className="title">{formTitle}</h1>
+            <FormComponent
+              isMember={isMember}
+              fieldsToRender={fildsToRender}
+            />
+            <button type="submit" onClick={this.onLinkClick} className="message">{message}</button>
```
Comment (GeFeSTTS) [inline src/client/containers/authentication/authentication.js]: There is not submit button. Better to use input in this case.

File: src/client/containers/authentication/authentication.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
+.authentication {
+    font-family: var(--font-authentication);
+    color: var(--authFont-white);
+    background-image: url('../../assets/images/authBackground.jpeg');
```
Comment (StanislavFedyna) [inline src/client/containers/authentication/authentication.css]: Compare these rules using the short form)

File: src/client/containers/authentication/authentication.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    }
+}
+
+@media screen and (max-width: 1000px) {
+    .authImage {
+        width: 60%;
+    }
+}
+
+@media screen and (max-width: 768px) {
```
Comment (NadiiaChorna) [inline src/client/containers/authentication/authentication.css:61]: From breakpoint 768px and lower it seems that your styles don't work correctly, please check them
image

File: src/client/containers/authentication/authentication.css (hunk 4 above)
Comment (Kindest13) [reply src/client/containers/authentication/authentication.css]: done

File: src/client/containers/formComponent/formComponent.css (hunk 2 above)
Comment (Kindest13) [reply src/client/containers/formComponent/formComponent.css]: done

File: src/client/containers/authentication/authentication.js (hunk 3 above)
Comment (Kindest13) [reply src/client/containers/authentication/authentication.js]: done

File: src/client/containers/authentication/authentication.js (hunk 1 above)
Comment (Kindest13) [reply src/client/containers/authentication/authentication.js]: done

File: src/client/app.css [hunk 6]
```diff
@@ -0,0 +1,14 @@
+@import url("https://fonts.googleapis.com/css?family=Orbitron&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Khand|Orbitron&display=swap");
+
+@import "./helpers/variables.css";
+
+* {
```
Comment (tonypntsrn) [inline src/client/app.css:6]: This can be added to body?

Comment (Kindest13) [reply src/client/app.css:6]: is that bad practic?because in that case it ruins styles of main page

**Review by StanislavFedyna — DISMISSED** [review]

**Review by NadiiaChorna — DISMISSED** [review]

File: src/client/containers/authentication/authentication.css (hunk 5 above)
Comment (Kindest13) [reply src/client/containers/authentication/authentication.css:61]: done, responsive for min height 360px

File: src/client/app.css (hunk 6 above)
Comment (tonypntsrn) [reply src/client/app.css:6]: It is a complex selector that can affect performance

**Review by tonypntsrn — APPROVED** [review]

**Review by NadiiaChorna — APPROVED** [review]

**Review by StanislavFedyna — APPROVED** [review]

**Pull request merged**